dali/operators/image/crop/bbox_crop.cc

```diff
@@ -96,191 +96,232 @@ struct Range {
 
-Value for ``min`` should satisfy ``0.0 <= min <= max``.
+The value for min should be greater than ``0.0``, and min should be less than or
+equal to the max value.
 
-Note: Providing ``aspect_ratio`` and ``scaling`` is incompatible with specifying `crop_shape`
-explicitly)code",
+.. note::
+  Providing ``aspect_ratio`` and ``scaling`` is incompatible when explicitly specifying the
+  ``crop_shape`` value.
+)code",
         std::vector<float>{1.f, 1.f})
     .AddOptionalArg(
         "ltrb",
-        R"code(If true, bboxes are returned as ``[left, top, right, bottom]``,
+        R"code(If set to True, bboxes are returned as ``[left, top, right, bottom]``;
 otherwise they are provided as ``[left, top, width, height]``.
 
-WARNING: This argument is deprecated. Use ``bbox_layout`` instead to specify the bbox encoding.
-E.g ``ltrb=True`` is equivalent to ``bbox_layout="xyXY"`` and ``ltrb=False`` corresponds to
-``bbox_layout="xyWH"``)code",
+.. warning::
+  This argument has been deprecated. To specify the bbox encoding, use ``bbox_layout`` instead.
+  For example, ``ltrb=True`` is equal to ``bbox_layout="xyXY"``, and ``ltrb=False`` corresponds
+  to ``bbox_layout="xyWH"``.
+)code",
         true)
     .AddOptionalArg(
         "num_attempts",
-        R"code(Number of attempts to get a crop window that matches the ``aspect_ratio`` and a selected
-value from ``thresholds``.
+        R"code(Number of attempts to get a crop window that matches the ``aspect_ratio`` and
+a selected value from ``thresholds``.
 
-After each ``num_attempts``, a different threshold will be picked, until either reaching a maximum of
-``total_num_attempts`` (if provided), or indefinitely otherwise.)code",
+After each ``num_attempts``, a different threshold will be picked, until the threshold reaches
+a maximum of ``total_num_attempts`` (if provided) or otherwise indefinitely.)code",
         1)
     .AddOptionalArg(
         "total_num_attempts",
-        R"code(If provided, it indicates the total maximum number of attempts to get a crop window that
-matches the ``aspect_ratio`` and any selected value from ``thresholds``.
+        R"code(If provided, it indicates the total maximum number of attempts to get a crop
+window that matches the ``aspect_ratio`` and any selected value from ``thresholds``.
 
-After ``total_num_attempts`` the best candidate will be chosen as a default.
+With the ``total_num_attempts`` value, the best candidate will be selected as a default.
 
-If not specified, the crop search will continue indefinitely until finding a valid crop.
+If this value is not specified, the crop search will continue indefinitely until a valid
+crop is found.
 
-WARNING: Not providing ``total_num_attempts`` could lead to an infinite loop if the provided arguments
-can't be statisfied.)code",
+.. warning::
+  If you do not provide a ``total_num_attempts`` value, this can result in an infinite
+  loop if the provided arguments cannot be satisfied.
```

Comment:
```suggestion
  If you do not provide a ``total_num_attempts`` value, this can result in an infinite
  loop if the conditions imposed by the arguments cannot be satisfied.
```

Reply:
Done